changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Test
+status: removed
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-02-07
+category: compute
+product: kubernetes
+---
+
+From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with cilium or `calico` as CNI. 
```

Comment:
From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with `cilium` or `calico` as CNI.